rest/server_context.go

```diff
@@ -266,6 +266,9 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 	pool := "default"
 	bucketName := config.Name
 
+	// Default value of _local document expiry
+	localDocExpirySecs := uint32(60 * 60 * 24 * 90) //90 days in seconds
```

Comment:
Can you move this default value to base/constants.go to make it easier to find/maintain?

Reply:
Moved to base/constants.go and renamed to DefaultLocalDocExpirySecs